server/controllers/TripController.js

```diff
@@ -1,29 +1,34 @@
 const TripModel = require('../models/TripModel')
 const uuid = require('uuid')
-const controllers = require('./Controllers')
 
 class TripController {
-  constructor() {}
+  constructor(stageController) {
+    this.stageController = stageController
```

Comment:
just wondering why you added the `this`, and do Andy and I need to do the same changes for `/users` and `/stages`

Reply:
if we instead import controllers and use it, then it will run into circular dependency issue. https://docs.nestjs.com/fundamentals/circular-dependency

Reply:
oh i see thanks :muscle: